tri.html

```diff
@@ -1,58 +1,61 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Triangle Game</title>
-    <link rel="stylesheet" type="text/css" href="tri.css">
-  </head>
-  <body>
-      
-    <h1>The <span id="titleDisplay">Triangle</span> Game</h1>
-
-    <div id="stripe">
-      <button class="btn" id="reset"></button>
-    </div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Triangle Game</title>
+  <link rel="stylesheet" type="text/css" href="tri.css">
+</head>
+
+<body>
+
+  <h1>The <span id="titleDisplay">Triangle</span> Game</h1>
+
+  <div id="stripe">
+    <button class="btn" id="reset"></button>
+  </div>
 
 
-    <div class="triangle">
-  
+  <div class="triangle">
+
     <div class="wrapper">
 
-    <div class="container1a" ></div>
+      <div class="container1a"></div>
 
-    <div class="container1" >
-         <div class="circle1" id="one">1</div>
-    </div>
-   
-    <div class="container2" >
-         <div class="circle2" id="two">2</div>
-         <div class="circle2" id="three">3</div>
-    </div>
-     
-    <div class="container3" >
-        <div class="circle3" id="four">4</div>
-        <div class="circle3" id="five">5</div>
-        <div class="circle3" id="six">6</div>
+      <div class="container1">
+        <div class="circle" id="one">1</div>
       </div>
-  
-      <div class="container4" >
-          <div class="circle4" id="seven">7</div>
-          <div class="circle4" id="eight">8</div>
-          <div class="circle4" id="nine">9</div>
-          <div class="circle4" id="ten">10</div>
+
+      <div class="container2">
+        <div class="circle" id="two" onclick="selectCircle(this)">2</div>
```

Comment:
Since all of your `onClick` events are the same, you could just create them dynamically by getting all `.circle` elements and assigning them the same click handler.

Reply:
Removed onClick from HTML, added event programmatically in JavaScript.